pkg/util/apiserver/oauth.go

```diff
@@ -252,14 +259,17 @@ func getKeyCloakClient() *retryablehttp.Client {
 
 // get the ca.crt from secret "<vz-env-name>-secret" in namespace "verrazzano-system"
 func getCACert(kubeClientSet kubernetes.Interface) []byte {
+	// Create log instance for initializing flags
+	logger := zerolog.New(os.Stderr).With().Timestamp().Str("kind", "CACert").Str("name", "Get").Logger()
```

Comment:
("kind", "CACert") and ("name", "Get") => we should remove these types of fields. The only field should be the Timestamp for now. As we discussed yesterday, we need to have the logging fields thought through.